app/models/custom_value/calculated_value_strategy.rb

```diff
@@ -51,6 +68,6 @@ def validate_type_of_value
   private
 
   def integer_value?
-    value && value =~ /\A\d+\z/
+    value =~ /\A[-+]?\d+\z/
```

Comment:
Looks like I didn't remember that `NilClass` implements `=~`, so `value &&` is indeed not needed